### PR TITLE
chore(DeploymentConfig): add goerli optimism network config

### PR DESCRIPTION
**Pull request by 0x-r4bbit**

### CODE REVIEW

File: packages/contracts/script/DeploymentConfig.s.sol [hunk 1]
```diff
@@ -42,11 +42,15 @@ contract DeploymentConfig is Script {
     address internal SNT_ADDRESS_MAINNET = 0x744d70FDBE2Ba4CF95131626614a1763DF805B9E;
     // solhint-disable-next-line var-name-mixedcase
     address internal SNT_ADDRESS_OPTIMISM_MAINNET = 0x650AF3C15AF43dcB218406d30784416D64Cfb6B2;
+    // solhint-disable-next-line var-name-mixedcase
+    address internal SNT_ADDRESS_OPTIMISM_GOERLI = 0xcAD273fA2bb77875333439FDf4417D995159c3E1;
```
Comment (felicio) [inline packages/contracts/script/DeploymentConfig.s.sol:46]: Non-blocking: will you be publishing and verifying the source code too https://goerli-optimism.etherscan.io/address/0xcAD273fA2bb77875333439FDf4417D995159c3E1#code.

File: packages/contracts/script/DeploymentConfig.s.sol [hunk 2]
```diff
@@ -112,6 +118,24 @@ contract DeploymentConfig is Script {
         });
     }
 
+    function getOptimismGoerliConfig() public returns (NetworkConfig memory) {
```
Comment (felicio) [inline packages/contracts/script/DeploymentConfig.s.sol:121]: Non-blocking: would remove `getMainnetConfig` and `getGoerliEthConfig` since not used.

**Review by felicio — APPROVED** [review]

File: packages/contracts/script/DeploymentConfig.s.sol (hunk 2 above)
Comment (0x-r4bbit) [reply packages/contracts/script/DeploymentConfig.s.sol:121]: Fair enough. Will do in a follow-up PR.

File: packages/contracts/script/DeploymentConfig.s.sol (hunk 1 above)
Comment (0x-r4bbit) [reply packages/contracts/script/DeploymentConfig.s.sol:46]: Need to look into this again, but it seems there's no API keys on testnet etherscans so have to check if verification is even enabled there.

File: packages/contracts/script/DeploymentConfig.s.sol (hunk 2 above)
Comment (0x-r4bbit) [reply packages/contracts/script/DeploymentConfig.s.sol:121]: https://github.com/status-im/community-dapp/pull/99

**Pull request merged**